pkg/virtctl/vmexport/vmexport_test.go

```diff
@@ -651,19 +649,16 @@ var _ = Describe("vmexport", func() {
 
 	Context("Manifest", func() {
 		const (
-			manifestUrl = "https://test.something.somewhere/test/all"
-			secretUrl   = "https://test.something.somewhere/test/secret"
+			manifestUrl = "/test/all"
+			secretUrl   = "/test/secret"
 		)
 
 		DescribeTable("should successfully create VirtualMachineExport if proper arg supplied", func(expected string, extraArgs ...string) {
-			vmexport.HandleHTTPRequest = func(_ kubecli.KubevirtClient, _ *exportv1.VirtualMachineExport, downloadUrl string, _ bool, _ string, headers map[string]string) (*http.Response, error) {
-				Expect(downloadUrl).To(Equal(manifestUrl))
-				Expect(headers).To(ContainElements(expected))
-				return &http.Response{
-					StatusCode: http.StatusOK,
-					Body:       io.NopCloser(strings.NewReader("data")),
-				}, nil
-			}
+			server.Config.Handler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				Expect(r.URL.String()).To(Equal(manifestUrl))
+				Expect(r.Header).To(HaveKeyWithValue("Accept", ConsistOf(expected)))
+				w.WriteHeader(http.StatusOK)
```

Comment:
`io.NopCloser(strings.NewReader("data"))` isn't needed?

Reply:
Empty body is sufficient to verify the expected URLs are accessed.